lib/wow_data/auth/db.ex

```diff
@@ -0,0 +1,30 @@
+defmodule WoWData.Auth.Db do
+  alias WoWData.Repo
+
+  def create(schema, module, attrs) do
+    schema
+    |> module.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def get(module, id \\ 1) do
+    data = Repo.get(module, id)
+
+    if data, do: {:ok, data}, else: {:error, :noent}
+  end
+
+  def update(module, new_value, id \\ 1) do
```

Comment:
Não adiciona números sequenciais como default, default apenas configurações que daria para ser utilizado em mais de um ambiente, e quando for adicionar configuração passa como opts

Reply:
Eu tive que colocar esse default por causa de uns problemas que eu tava tendo com os testes. Acho que vou mudar a tabela de access_token pra configurations ou coisa do tipo e deixar o nome da config (access token no caso) como primary key, o que vc acha?

Reply:
Eu prefiro deixar como PK apenas inteiros seguencias

Reply:
Que tipo de problemas?